conan/tools/files/__init__.py

```diff
@@ -1,5 +1,6 @@
 from conan.tools.files.files import load, save, mkdir, ftp_download, download, get, rename, \
-    load_toolchain_args, save_toolchain_args, chdir
+    load_toolchain_args, save_toolchain_args, chdir, absolute_to_relative_symlinks, \
```

Comment:
maybe group all symlinks tools in some ``conan.tools.files.symlinks`` or something?

Reply:
yeah, I thought about it, let's do it